File: Java/String-1/src/firstHalf/FirstHalf.java
package firstHalf;

public class FirstHalf {
    //Retorne uma String que seja a metade da String dada, não há limite de tamanho para a String;

    public String firstHalf(String str) {
        int tamanho = str.length()/2;
        return str.substring(0,tamanho);
    }
}

File: Python/String-1/first_half.py
"""
Dada uma String de tamanho par "str", retorne a sua primeira metade.
Ex.:(('WooHoo') → 'Woo'; ('abcdef') → 'abc'; ('HelloThere') → 'Hello').
"""


def first_half(str):
    return str[:int(len(str)/2)]


print(first_half("WooHoo"))
